packages/spin/spin.0.5.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis:
+  "Project scaffolding tool and set of templates for Reason and OCaml"
+description: """
+Project scaffolding tool and set of templates for Reason and OCaml
+"""
+maintainer: ["Thibaut Mattio"]
+authors: ["Thibaut Mattio"]
+license: "MIT"
+homepage: "https://github.com/tmattio/spin"
+doc: "https://tmattio.github.io/spin"
+bug-reports: "https://github.com/tmattio/spin/issues"
+depends: [
+  "ocaml" {>= "4.06.0" & < "4.10.0"}
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+  "reason" {build}
```

Comment:
I can't find any use of `reason` in the source code. Is this really needed?

Reply:
The codebase is in Reason, so this is needed to build the project.

Reply:
indeed. My bad ^^